app/controllers/circle/projects_controller.rb

```diff
@@ -61,6 +61,7 @@ def update
 
   def complete
     authorize! :complete, current_project
+    close_all_tasks_supplies
```

Comment:
Hi Nolan, no big deal, I moved the code and merged your PR. Thanks!

Btw. another detail: you used `#map` for `current_project.tasks.incomplete.map`; `#each`is enough there since all you want to do is iterate over the collection; the return values of each command in the block (which only map gives you) are not needed.